### ISSUE TITLE
--debug doesn't work

### ISSUE THREAD

**Issue by area** [ethereumjs/testrpc#63]
If I try and run with the --debug flag, then on deployment (with truffle) I get

```
     Error deploying Contract:

     Error: VM Exception while executing transaction: ReferenceError: logger is not defined
         at /Users/user/.node/lib/node_modules/ethereumjs-testrpc/lib/blockchain.js:506:18
```

**Comment by tcoulter** [comment]
Good catch. Will fix today.

**Comment by tcoulter** [comment]
Fixed and published to npm: v2.0.2.

Thanks!
